From 811f58acf65fc430ac9f2da42c4570f07fcf5cd8 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 14 Apr 2017 15:49:37 +0800
Subject: [PATCH 344/347] arm: Ignore the pkey system calls for now

Eliminates warning messages:

warning: #warning syscall pkey_mprotect not implemented [-Wcpp]
warning: #warning syscall pkey_alloc not implemented [-Wcpp]
warning: #warning syscall pkey_free not implemented [-Wcpp]

Hopefully we will remember to revert this commit if we ever implement
them.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/include/asm/unistd.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/include/asm/unistd.h b/arch/arm/include/asm/unistd.h
index 194b699..9d49287 100644
--- a/arch/arm/include/asm/unistd.h
+++ b/arch/arm/include/asm/unistd.h
@@ -51,5 +51,8 @@
  */
 #define __IGNORE_fadvise64_64
 #define __IGNORE_migrate_pages
+#define __IGNORE_pkey_mprotect
+#define __IGNORE_pkey_alloc
+#define __IGNORE_pkey_free
 
 #endif /* __ASM_ARM_UNISTD_H */
-- 
1.7.5.4

